.editorconfig

```diff
@@ -0,0 +1,18 @@
+# WordPress Coding Standards
+# https://make.wordpress.org/core/handbook/coding-standards/
+	
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = tab
+
+[{.jshintrc,*.json,*.yml}]
```

Comment:
@westonruter Seems just right. Pushed https://github.com/xwp/wp-dev-lib/pull/194/commits/49484a476f01228fdefa00f3b9c18a277f6a2061 to amend.